src/osemgrep/language_server/Test_LS_e2e.ml

```diff
@@ -1024,24 +1025,28 @@ let test_ls_libev () =
 (* Entry point *)
 (*****************************************************************************)
 
+module Test = Alcotest_ext
```

Comment:
you should do this let t = Alcotest_ext.create also here

Reply:
Also, we should find a shorter name for Alcotest_ext. Candidates include Testo ("Party hard, test harder" - DJ Testo) and Protest.